packages/ui/hooks/fuse/usePoolDetails.ts

```diff
@@ -14,14 +17,28 @@ export const usePoolDetails = (assets: NativePricedFuseAsset[] | undefined) => {
           mostSuppliedAsset = asset;
         }
         if (
-          convertMantissaToAPY(asset.supplyRatePerBlock, 365) >
-          convertMantissaToAPY(topLendingAPYAsset.supplyRatePerBlock, 365)
+          convertMantissaToAPY(
+            asset.supplyRatePerBlock,
+            getBlockTimePerMinuteByChainId(currentChain.id),
+            365
+          ) >
+          convertMantissaToAPY(
+            topLendingAPYAsset.supplyRatePerBlock,
+            getBlockTimePerMinuteByChainId(currentChain.id),
+            365
```

Comment:
we could maybe make this optional and the default value 356?

Reply:
> 



> Hm how about we move these functions also to the SDK, @smartdevcom? We maybe even could calculate these values before and just put them also in the `PoolData`? And also add some happy path tests for some values. To just make sure they are correct.
> 
> `convertMantissaToAPY` `convertMantissaToAPR`
> 
> I think this could be helpful.

Sure, I will do it